Themes/default/scripts/jquery.sceditor.smf.js

```diff
@@ -145,7 +152,7 @@
 						adjheight = closeButton.height() + titlebar.height();
 						$dropdown = $('<div class="centerbox sceditor-smileyPopup">')
 							.append(popupContent)
-							.appendTo($('.sceditor-container'));
+							.appendTo($('center'));
```

Comment:
this a tag selector? What happens if more than one exist?

Reply:
I guess
`.sceditor-insertemoticon center[unselectable="on"]` would be better?

Reply:
don't need the attribute selector, the more specific selector will only ever return one element

```suggestion
							.appendTo($('.sceditor-insertemoticon center'));
```

Reply:
Cool, will update this weekend with some changes.